packages/next/src/migrations/update-14-5-7/update-next-eslint.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { readJson, Tree } from '@nrwl/devkit';
+import { createTreeWithEmptyV1Workspace } from '@nrwl/devkit/testing';
+import { updateNextEslint } from './update-next-eslint';
+
+describe('Add next eslint 14.5.7', () => {
+  let tree: Tree;
+
+  beforeEach(async () => {
+    tree = createTreeWithEmptyV1Workspace();
```

Comment:
I think we want to use `createTreeWithEmptyWorkspace` here, and `updateProjectConfiguration` so it's more future-proof.